CMakeLists.txt

```diff
@@ -157,9 +163,9 @@ separate build directory. Note: Please remove autogenerated file \
 `CMakeCache.txt` and directory `CMakeFiles` in the current directory.")
 endif()
 
-if(SLEEF_ENABLE_LTO AND BUILD_SHARED_LIBS)
-  message(FATAL_ERROR "SLEEF_ENABLE_LTO and BUILD_SHARED_LIBS cannot be specified at the same time")
-endif(SLEEF_ENABLE_LTO AND BUILD_SHARED_LIBS)
+if(SLEEF_ENABLE_LTO AND _INTERNAL_SLEEF_BUILD_SHARED_LIBS)
+  message(FATAL_ERROR "SLEEF_ENABLE_LTO and _INTERNAL_SLEEF_BUILD_SHARED_LIBS cannot be specified at the same time")
+endif(SLEEF_ENABLE_LTO AND _INTERNAL_SLEEF_BUILD_SHARED_LIBS)
 
 if(SLEEF_ENABLE_LTO)
   cmake_policy(SET CMP0069 NEW)
```

Comment:
```suggestion
```
That policy is already enabled by `cmake_minimum_required(VERSION 3.18)`. It's enabled for 3.9+.